src/representation/dmp/src/DMPModel.h

```diff
@@ -44,13 +44,12 @@ class DMPModel{
     /**
      * \return true if the model is consistent, false otherwise
      */
-    virtual bool is_valid() const;
+    bool is_valid() const;
 
     void to_yaml_file(std::string filepath);
 
     friend std::ostream& operator << (std::ostream& os, DMPModel& val);
 
-PRIVATE:
```

Comment:
If that is the only occurence of PRIVATE in this file, you can remove the macro at the top of the file.

Reply:
ok